File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        keyValues = {}
        used = [False] * len(nums)
        for i in range(len(nums)):
            counter = 1
            if used[i]:
                continue
            used[i] = True
            for j in range(i+1, len(nums)):
                if nums[i] == nums[j]:
                    used[j]= True
                    counter += 1
            keyValues[nums[i]] = counter
        sorted_items = sorted(keyValues.items(), key=lambda x: x[1], reverse=True)
        result = [item[0] for item in sorted_items[:k]]
        return result

File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
    let result = 0;
    for (let i = 0; i < nums.length; i++) {
        for (let j = 0; j < nums.length; j++){
            if(nums[i] == nums[j]){
                result++;
            }
        }
    }
     if (result == nums.length){
        return false;
    } else {
    return true;
    }
   
    }
}
